docs/contributors/demo-page-css.md

```diff
@@ -9,4 +9,9 @@ feel free to preview config files:
 - [postcss.config.js](postcss.config.js)
 - [tailwind.config.js](tailwind.config.js)
 
-TODO: livereload
+rollup build is updated to handle livereload, there are 2 scripts in [package.json](package.json) that permits that: \
+1. Build the output tailwind.css file. \
+```"demo:css": "postcss src/static/css/styles.css -o src/static/css/tailwind.css"```
+
+2. Watch for the changes in html and styles.css. \
+```"watch:css": "chokidar \"src/static/css/styles.css\" \"src/*.html\" --command \"npm run demo:css\" --verbose"```
```

Comment:
I propose to not describe the scripts to simplify maintenance; contributors have the script names so they can go to the `package.json` file if they are curious
```suggestion
The Rollup build is updated to handle livereload, there are 2 scripts in [package.json](../../package.json) that permits that: \
1. Build the output tailwind.css file: `demo:css`

2. Watch for the changes in html and styles.css. `watch:css`
```